controllers/topology/reconcile_state.go

```diff
@@ -372,7 +401,7 @@ func (r *ClusterReconciler) reconcileReferencedTemplate(ctx context.Context, in
 	// Set up a cleanup func for removing the old template.
 	// NOTE: This function must be called after updating the object containing the reference to the Template.
 	return func() error {
-		log.Info("Deleting", in.desired.GroupVersionKind().String(), in.desired.GetName())
+		log.Info(fmt.Sprintf("Deleting %s/%s", in.current.GroupVersionKind().String(), in.current.GetName()))
```

Comment:
Why using fmt and not the variable like above?

Reply:
This applies to all the other log lines as well.

Reply:
I think our problem is that there are at least the following frequently used ways to consume logs: (and we cannot 100% optimize for both at the same time)

* "plain text" via e.g. kubectl: there it should look as easily readable as possible in the format it's logged
* logging UIs like Kibana: here the structured logs are parsed first. It's ideal to be able to filter based on key/value pairs, e.g. to be able to filter all log lines corresponding to a certain object would be great.

I don't have a strong opinion either way, just wanted to provide some additional input/context.